tools/queries.cpp

```diff
@@ -152,7 +163,7 @@ void perftest(const std::string &index_filename,
         std::ifstream tin(*thresholds_filename);
         size_t idx = 0;
         while (std::getline(tin, t)) {
-            thresholds[idx] = std::stof(t);
+            thresholds[idx] = std::stof(t) - 0.0001;
```

Comment:
Maybe we should fix the thresholds instead. 
I propose to get rid of `- 0.0001` here and fix the way we compute thresholds in a separate PR.